packages/cli/test/fixtures/copyright/single-package/index.js

```diff
@@ -1,7 +1,7 @@
-// Copyright IBM Corp. and LoopBack contributors 2020. All Rights Reserved.
-// Node module: @loopback/cli
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
+// Copyright ACME Inc. 2020,2023. All Rights Reserved.
```

Comment:
@aaqilniz, could you please revert this change?

Reply:
Hi, @dhmlau. The statements were mistakenly modified. I have reverted the changes.